gnd/src/main/java/com/google/android/gnd/model/layer/Style.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.gnd.model.layer;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class Style {
```

Comment:
Very nice, leaving a comment here to capture my thoughts for posterity--It'd be nice if we can make this generic enough to support the other cases where we'll need styling too (e.g. tile selections)